src/grid/cell.component.ts

```diff
@@ -192,6 +189,13 @@ export class IgxGridCellComponent {
         this.handleInlineEditMode(event);
     }
 
+    public update(val: any) {
```

Comment:
I guess changing a `value` setter to a modifier method introduces a breaking change. If this affects any of the documentation, readmes, etc. please pull updates to the related documentation. Check the getting-started page as well, for any code that may stop working.